internal/unix/error.go

```diff
@@ -0,0 +1,21 @@
+package unix
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/cilium/ebpf/internal"
+)
+
+func errNonLinux() error {
```

Comment:
Could you add a docstring to this? Hard to sight-read without looking up documentation.